sentinelsat/__init__.py

```diff
@@ -2,12 +2,9 @@
 
 # Import for backwards-compatibility
 from . import sentinel
-
+from .exceptions import *
```

Comment:
Perhaps be explicit about the imports here, even though we are just in `__init__.py`